<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>681</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>90</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>普通素性检测</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>输入检测数</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>151</width>
       <height>18</height>
      </rect>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Miller-Rabin素性检测</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>681</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>输入两个大素数</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>160</y>
       <width>261</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>160</y>
       <width>261</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>输入明文</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>密文</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>140</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>明文</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>110</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>加密</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>681</width>
       <height>211</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLineEdit" name="lineEdit_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_9">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>50</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>输入两个大素数</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_10">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_11">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_12">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>160</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>输入明文</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>密文</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>140</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>明文</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>110</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>加密</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>110</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>解密</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>681</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
